webhook/webhook.go

```diff
@@ -452,6 +454,9 @@ func (ac *AdmissionController) ServeHTTP(w http.ResponseWriter, r *http.Request)
 		http.Error(w, fmt.Sprintf("could encode response: %v", err), http.StatusInternalServerError)
 		return
 	}
+
+	// Only report valid requests
+	ac.StatsReporter.ReportRequest(review.Request, response.Response, time.Since(ttStart))
```

Comment:
This causes a NPE. StatsReporter is new and nil in all repos.

Reply:
Ah yes, that makes sense. I was pulling this in and making the changes in the serving repo to initialize this correctly, but I forgot this was a common package :/

I'll push a fix out in a couple of hours or roll it back.